functions/.eslintrc.json

```diff
@@ -118,6 +118,27 @@
     "promise/catch-or-return": 2,
 
     // Warn against nested then() or catch() statements
-    "promise/no-nesting": 1
+    "promise/no-nesting": 1,
+
+
+    // Custom rules for MealMatch project:
```

Comment:
ooh that's why they are red! lol I'll just leave it since this file was generated by firebase when i initiated the cloud functions, and it had comments like that already. Thanks for pointing it out tho!